spec/aptible/auth/token_spec.rb

```diff
@@ -82,7 +82,7 @@
     end
 
     describe '#authenticate_user' do
-      let(:args) { %w(user@example.com foobar) }
+      let(:args) { %w[user@example.com foobar] }
```

Comment:
Interesting - is there a difference here? I get the following in my console:
```
2.5.1 :003 > %w(user@example.com foobar) == %w[user@example.com foobar]
 => true
```

Reply:
It was a rubocop complaint. I think they're equivalent